hedera-node/configuration/compose/settings.txt

```diff
@@ -1,15 +1,18 @@
-checkSignedStateFromDisk, 1
-csvAppend, 0
-csvFileName, PlatformStats
-dbBackup.active, 0
-dbConnection.host, hedera-postgres
-dbConnection.schema, fcfsNODE_ID
-doUpnp, 0
-freezeSettings.active, 0
-maxOutgoingSyncs, 2
-showInternalStats, 1
-state.roundsExpired, 500
-state.roundsStale, 25
-state.saveStatePeriod, 300
-state.signedStateDisk, 10
-throttle7extra, 0.2
+dbBackup.active,                               0
+dbConnection.host,                             hedera-postgres
+dbConnection.schema,                           fcfsNODE_ID
+checkSignedStateFromDisk,                      1
+csvFileName,                                   MainNetStats
+dbBackup.active,                               1
```

Comment:
`dbBackup.active` is repeated but with different value. Can be fixed in another PR.